src/citrine/informatics/predictors/generalized_mean_property_predictor.py

```diff
@@ -55,7 +55,7 @@ class GeneralizedMeanPropertyPredictor(
 
     input_descriptor = _properties.Object(FormulationDescriptor, 'config.input')
     properties = _properties.List(_properties.String, 'config.properties')
-    p = _properties.Float('config.p')
+    p = _properties.Integer('config.p')
```

Comment:
Should we change this on the backend also, or leave is as a float in case we want to allow non-integer powers in the future?

Reply:
I left a comment on the ticket: https://citrine.atlassian.net/browse/PLA-6448. I think that we should leave it as a float and not modify the backend.

Reply:
How do you feel about changing it here? It seems strange to me that we would type is as a `float` in Python but throw a validation error if it's anything but an `int`. However the mismatch between Python and the backend is weird too. I _think_ this is clearer from a user's perspective, but I could see the argument for leaving the type as-is.

Reply:
Yes, sorry, I think you're right that we should change it to an Integer here. But we should leave it as a Double in the backend. I don't think that will cause deser issues, since everything that has been persisted so far has been an integer.